install_config/registry/securing_and_exposing_registry.adoc

```diff
@@ -268,8 +268,7 @@ DNS to the router's IP address.
 +
 [NOTE]
 ====
-Passthrough is currently the only type of route supported for exposing the
-secure registry.
+Reencrypt routes are also supported for exposing the secure registry.
```

Comment:
s/Reencrypt/Re-encrypt
(we seem to use the hyphenated version)

Reply:
fixed